.travis.yml

```diff
@@ -9,9 +9,6 @@ cache:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - EXECUTE_CS_CHECK=true

```

Comment:
CS check needs to be moved under php7, otherwise no check at all will be executed..